src/views/connected/structure/MaStructure.js

```diff
@@ -76,7 +76,7 @@ function MaStructure() {
       dispatch(
         alerteEtSpinnerActions.getMessageAlerte({
           type: 'error',
-          message: errorMessage[0],
+          message: errorMessage[0] || 'Une erreur s’est produite',
```

Comment:
ca va pas péter sur error message not array dans le errorMessage[0] ?

Reply:
oui finalement j'ai isolé l'erreur au chargement de la structure